src/configuration.js

```diff
@@ -12,24 +12,29 @@ import {
 } from './util';
 
 /**
- * Defaults configuration.
+ * Default configuration.
```

Comment:
original meaning was "a configuration object that serves default values", hence the plural tense. If we change it, it should be to something like "A configuration for the various SDK functions"...